content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md

```diff
@@ -29,11 +29,9 @@ Feature flag tracking is available in the RUM Browser SDK. To start, set up [RUM
 <details>
   <summary>Before <code>v5.17.0</code></summary>
 
-If you are using a version anterior to 5.17.0, initialize the RUM SDK and configure the `enableExperimentalFeatures` initialization parameter with ` ["feature_flags"]` to start collecting feature flag data.
-
-{{< tabs >}}
-{{% tab "NPM" %}}
+If you are using a version previous to 5.17.0, initialize the RUM SDK and configure the `enableExperimentalFeatures` initialization parameter with `["feature_flags"]` to start collecting feature flag data.
 
+{{% collapse-content title="NPM" level="h4" %}}
```

Comment:
**Question**: any reason why you changed the `tabs` to `collapse-content`? We usually use tabs for this (ex: https://docs.datadoghq.com/real_user_monitoring/browser/advanced_configuration/?tab=npm#override-default-rum-view-names)

Reply:
**Answer**: Yes; as mentioned in my [comment](https://github.com/DataDog/documentation/pull/25823#issuecomment-2427482620), I changed the tabs because they're nested within another set of tabs. This leads to unexpected behavior and formatting on the documentation site. 

For example, you can see from this screenshot with nested tabs how the nested options (`NPM`, `CDN async`, `CDN sync`) are also added to the **Setup** tabs:
<img width="740" alt="nested_tabs" src="https://github.com/user-attachments/assets/2a713b02-6b72-4cd8-99cb-6c858abcd163">

I'm happy to keep iterating on this if you'd like to present the information differently. Thanks in advance for any feedback!

Reply:
Aah I missed the nested tabs. Looks good then!

Reply:
I cannot approve the PR as "PR authors can't approve their own PR". I will ask a teammate for the browser-sdk review.